From 6f5fcf6b8c9754b5e9e650c7017ef5a77446687e Mon Sep 17 00:00:00 2001
From: Dave Mankoff <mankoff@google.com>
Date: Mon, 15 Nov 2021 19:21:44 +0000
Subject: [PATCH 73/84] [Q_asb_2022-02] RESTRICT AUTOMERGE Remove line of code
 that was mistakently left in.

This line was removed in O, S, & P, but somehow survived in the Q and R branches.

Bug: 193444889
Merged-In: I56589865427b10e2eab68e1ed2e7c290572a9edc
Change-Id: I56589865427b10e2eab68e1ed2e7c290572a9edc
(cherry picked from commit d7097b817bb5a661e4bbc22829f6dc0a6ddca27c)
Merged-In: I56589865427b10e2eab68e1ed2e7c290572a9edc
---
 .../com/android/systemui/shared/plugins/PluginManagerImpl.java   | 1 -
 1 file changed, 1 deletion(-)

diff --git a/packages/SystemUI/shared/src/com/android/systemui/shared/plugins/PluginManagerImpl.java b/packages/SystemUI/shared/src/com/android/systemui/shared/plugins/PluginManagerImpl.java
index fb109d2bda5b..672c0ae822f2 100644
--- a/packages/SystemUI/shared/src/com/android/systemui/shared/plugins/PluginManagerImpl.java
+++ b/packages/SystemUI/shared/src/com/android/systemui/shared/plugins/PluginManagerImpl.java
@@ -196,7 +196,6 @@ public class PluginManagerImpl extends BroadcastReceiver implements PluginManage
         filter.addAction(DISABLE_PLUGIN);
         filter.addDataScheme("package");
         mContext.registerReceiver(this, filter, PluginInstanceManager.PLUGIN_PERMISSION, null);
-        mContext.registerReceiver(this, filter);
         filter = new IntentFilter(Intent.ACTION_USER_UNLOCKED);
         mContext.registerReceiver(this, filter);
     }
-- 
2.35.1

